src/Sylius/Component/Shipping/Resolver/CompositeMethodsResolver.php

```diff
@@ -45,4 +45,12 @@ public function getSupportedMethods(ShippingSubjectInterface $shippingSubject)
 
         return [];
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(ShippingSubjectInterface $subject)
+    {
+        return true;
```

Comment:
`ShippingMethodsByZonesAndChannelResolver` seems to be the only class which implements this interface that does not return true. Could there be a possibility that a separate interface is required for that class and `supports()` can be removed from the `MethodResolverInterface`? Not sure if I'm over thinking this as I've only just started diving into Sylius (just looking at some PR's to get the feel of it)

https://github.com/Sylius/Sylius/blob/db5a8dcd3b23fbaea73deaf80ddc6487ea5b60b9/src/Sylius/Component/Core/Resolver/ShippingMethodsByZonesAndChannelResolver.php